input/pagecontent/changes.md

```diff
@@ -6,6 +6,7 @@
 This change log documents the significant updates and resolutions implemented from version 0.3.0-ballot to the upcoming August ballot.
 
 #### Changes in this version
+- added AUCDI element mapping Date/Time of Measurement and Date/Time of Observation, and added future consideration note for Last Updated
```

Comment:
The sentence should start with an uppercase letter, please change

Reply:
fixed